test/e2e/data/cni/cilium/cilium-no-kubeproxy.yaml

```diff
@@ -0,0 +1,1056 @@
+---
```

Comment:
pls document the location from where we got this yaml

Reply:
@deepakm-ntnx It's documented inline: https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/310/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R57